src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final String defaultColor = Color.WHITE.name();
+    private final double defaultRadius = 10;
+    private final int maxValues = 6;
```

Comment:
those values should be constants
```suggestion
    private static final String DEFAULT_COLOR = Color.WHITE.name();
    private static final double DEFAULT_RADIUS = 10;
    private static final int MAX_VALUE = 6;
```